docs/index.rst

```diff
@@ -35,6 +42,11 @@ http://higlass.io/version.txt
 .. |higlass-docker| replace:: **higlass-docker**
 .. _higlass-docker: https://github.com/higlass/higlass-docker
 
+|higlass-python|_ - This package contains directives to locally visualize data with HiGlass in `Jupyter notebooks or Lab <https://jupyter.org>`_.
```

Comment:
Let's place this above `higlass-docker` and also stick in a section for `clodius`: e.g. "this package implements the aggregation and tile generation logic for various data types".

Reply:
Done!